src/OptinForms/AbstractOptinTheme.php

```diff
@@ -1121,6 +1121,41 @@ public function shortcode_optin_form_custom_fields($atts)
                             }
                             $html .= "</select>" . $atts['tag_end'];
                             break;
+                        case 'country':
+                            $display_type = moVar($field, 'country_field_options', 'alpha_two', true);
+
+                            $html .= $atts['tag_start'];
+
+                            switch ($display_type) {
+                                case 'alpha-2':
+                                    $countries_lists = \MailOptin\Core\countries_array();
+                                    if (is_array($countries_lists) && ! empty($countries_lists)) {
+                                        $placeholder = ! empty($placeholder) ? $placeholder : esc_html__('Select...', 'mailoptin');
+                                        $html        .= "<select name=\"$field_id\" $data_attr class=\"$class\" id=\"$id\" style=\"$style\">";
+                                        $html        .= "<option value='' selected='selected'>$placeholder</option>";
+
+                                        foreach ($countries_lists as $key => $countries_list) {
+                                            $html .= sprintf('<option value="%1$s">%2$s (%1$s)</option>', $key, $countries_list);
```

Comment:
@ShoboySnr this is how sprintf named/index parameter works. Notice there is no ending $key variable again?

Reply:
Great.. thank you, understood